supplementary/alica_ros1/supplementary_tests/src/test/test_alica_dynamicloadcreator.cpp

```diff
@@ -146,7 +146,7 @@ TEST_F(AlicaDynamicLoading, simple_transition_condition_load)
     }
 
     ASSERT_EQ(conditionModel->getName(), "VariableHandlingStart");
-    TransitionConditionContext ctx{conditionModel->getName(), conditionModel->getLibraryName(), 0, 0};
+    TransitionConditionContext ctx{conditionModel->getName(), conditionModel->getLibraryName(), 0};
```

Comment:
Does this break API?  We might need to make a deprecated constructor

Reply:
No it does not change anything with the API, the condition context is created internally only. I also run lbc_tests and sootballs_alica_base, both run without any changes